qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_powder_tof/xml_dump/xml_dump_widget.py

```diff
@@ -0,0 +1,30 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+"""
+DNS Path widget.
+"""
+
+from mantidqtinterfaces.dns_powder_tof.data_structures.dns_widget import \
+    DNSWidget
+from mantidqtinterfaces.dns_powder_tof.xml_dump.xml_dump_model import \
+    DNSXMLDumpModel
+from mantidqtinterfaces.dns_powder_tof.xml_dump.xml_dump_presenter import \
+    DNSXMLDumpPresenter
+from mantidqtinterfaces.dns_powder_tof.xml_dump.xml_dump_view import \
+    DNSXMLDumpView
+
+
+class DNSXMLDumpWidget(DNSWidget):
```

Comment:
I am not seeing this widget in the GUI, it is enabled currently?

Reply:
All widgets (children of the DNSWidget class) are considered as “containers” for presenter, model and view. The view of xml_dump is inherited from the DNSReductionGUIView class that loads dns_gui_main_reduced_menu.ui. The xml_dump then handles saving the GUI’s status to an xml document, as well as to subsequent loading of the file in order to save some time for a user.